rootex/common/types.h

```diff
@@ -96,7 +96,12 @@ using Optional = std::optional<T>;
 #include <vector>
 /// std::vector
 template <class T>
-using Vector = std::vector<T>;
+using Vector = std::vector<T>;
+
+#include <unordered_set>
+/// std::unordered_set
+template <class T>
+using Unordered_set = std::unordered_set<T>;
```

Comment:
Does this code even compile? `std::unordered_set` requires 2 template arguments.

Reply:
Don't we only need to provide the Key type and the rest (hash, key_equal, allocator) are picked as the default ones?
ref: https://en.cppreference.com/w/cpp/container/unordered_set

Reply:
My bad, confused this one with an unorderd_map. You can resolve this conversation